web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/selectors/CSSSelectors.kt

```diff
@@ -35,31 +35,21 @@ sealed interface Nth {
 }
 
 abstract class CSSSelector internal constructor() {
-    override fun equals(other: Any?): Boolean {
-        return this === other || asString() == (other as? CSSSelector)?.asString()
-    }
 
-    internal open fun contains(other: CSSSelector, strict: Boolean = false): Boolean {
-        return if (strict) this === other else this == other
+    internal open fun contains(other: CSSSelector): Boolean {
+        return this === other
```

Comment:
Just to understand the motivation to remove `strict`.

We never passed `true` for it? right? so it's a redundant check in 100% cases?